### ISSUE TITLE
RTX 2080s performs better than RTX 3080 in Semantic Segmentation reference process,why?

### ISSUE THREAD

**Issue by afpapqy** [pytorch/pytorch#52570]
### Please note that this issue tracker is not a help form and this issue will be closed.

We have a set of [listed resources available on the website](https://pytorch.org/resources). Our primary means of support is our discussion forum:

- [Discussion Forum](https://discuss.pytorch.org/)

**Comment by peterjc123** [comment]
1. Please add `torch::cuda::synchronize()` before/after `forward` calls.
2. For CUDA 11.0, it only supports arch 8.0. But RTX3080 is of arch 8.6, so you'll need to use the binaries built with CUDA >= 11.1 to get optimized performance.

**Comment by afpapqy** [comment]
> 1. Please add `torch::cuda::synchronize()` before/after `forward` calls to get correct timings.
> 2. For CUDA 11.0, it only supports arch 8.0. But RTX3080 is of arch 8.6, so you'll need to use the binaries built with CUDA >= 11.1 to get optimized performance.

Thanks！
But is there a available version of pytorch/libtorch which is corresponding to cuda11.1?
I didn't find one in `https://pytorch.org/`
![image](https://user-images.githubusercontent.com/33414577/108716491-29462700-7557-11eb-83e6-c5109901cdaa.png)

**Comment by peterjc123** [comment]
Nope, but it will be very soon for a stable release. Alternatively, you may try out the nightlies.

**Comment by afpapqy** [comment]
> Nope, but it will be very soon for a stable release. Alternatively, you may try out the nightlies.

Wow, that's really something to look forward to. 
Before that come out, which single-GPU&CUDA combination do you think has the best performance?  ^_^

**Comment by peterjc123** [comment]
It's hard to say since you said best. I can only only tell which comb is not so good.😂

**Comment by afpapqy** [comment]
> It's hard to say since you said best. I can only only tell which comb is not so good.😂

Hhh, that's true.
Anyway,thanks for your reply..:neckbeard:

**Comment by ngimel** [comment]
Please provide a reproducible script demonstrating regression, preferably without outside dependencies like OpenCV. cc @ptrblck.
